frontend/packages/console-plugin-sdk/src/withExtensions.tsx

```diff
@@ -25,7 +25,7 @@ import { Extension, ExtensionTypeGuard } from './typings';
  *   isPerspective,
  * } from '@console/plugin-sdk';
  *
- * const Example = withExtensions<NavItem | Perspective, ExampleExtensionProps>({
+ * const Example = withExtensions<ExampleExtensionProps>({
```

Comment:
We need to differentiate between "ExtensionProps" and "OwnProps" to ensure we get the latter on the output.